rebar.lock

```diff
@@ -17,6 +21,7 @@
  {<<"gun">>,{pkg,<<"gun">>,<<"1.3.2">>},0},
  {<<"meck">>,{pkg,<<"meck">>,<<"0.8.13">>},0},
  {<<"p1_utils">>,{pkg,<<"p1_utils">>,<<"1.0.18">>},1},
+ {<<"proper">>,{pkg,<<"proper">>,<<"1.3.0">>},1},
```

Comment:
Do we need Proper here?

Reply:
Not really, that's a mixed dep of a dep, getting that fixed asap 😄